src/clj/rems/api/workflows.clj

```diff
@@ -47,6 +50,21 @@
       (assoc (format-workflow wf)
         :actors (db/get-workflow-actors {:wfid (:id wf)})))))
 
+(defn create-workflow [{:keys [prefix title rounds]}]
```

Comment:
This should allow to create workflows that are automatically approved after applying for a resource